### PR TITLE
Fix footer in darkmode  (Lombiq Technologies: NEST-515)

### PR DESCRIPTION
**Pull request by Psichorex**
The Dark Theme currently leaves the Footer section's background color with `#f5f5f5` which is barely readable.
It is also statically defined in `_layout.scss` thus it wasn't coming from Bootstrap.

d0334172-4625-4915-ae63-92de248ebe98

Quick change to handle the footer's background color so that it matches with the selected theme.

image

### CODE REVIEW

**Review by hishamco — APPROVED** [review]

**Comment by Piedone** [comment]
@hishamco please merge PRs, especially for non-maintainers who can't merge their own, after an approve (unless you're waiting for second opinions, of course, but then comment this explicitly and add the `needs triage` label).

**Pull request merged**
